storage/src/tests/distributor/bucketdbupdatertest.cpp

```diff
@@ -1733,7 +1775,7 @@ BucketDBUpdaterTest::testPendingClusterStateWithGroupDownAndNoHandover()
 
     // Group is down, but config says to not do anything about it.
     CPPUNIT_ASSERT_EQUAL(
-            std::string(""),
+            getNodeList({0, 1, 2, 3, 4, 5}, _bucketSpaces.size() - 1),
```

Comment:
I presume this change is due to the global bucket space's distribution config which differs in behavior from the default space?